wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageCellTableViewAdapter.swift

```diff
@@ -183,7 +183,7 @@ class ConversationMessageCellTableViewAdapter<C: ConversationMessageCellDescript
     }
 
     func showSecuredMenu(for text: String) {
-        let actions = [MessageAction.visitLink(text), MessageAction.reply, MessageAction.edit, MessageAction.openDetails, MessageAction.delete, MessageAction.cancel]
```

Comment:
Since we removed `text` from visitLink can we also remove it from the `showSecuredMenu`?

Reply:
fixed